confidant/models/credential.py

```diff
@@ -145,3 +148,30 @@ def _get_decrypted_credential_pairs(self):
     @property
     def decrypted_credential_pairs(self):
         return(self._get_decrypted_credential_pairs())
+
+
+class CredentialArchive(CredentialBase):
```

Comment:
You might want an archive_at datetime field for auditing purposes

Reply:
Good point. Let me add that.